packages/a11y/docs/VisuallyHidden.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+import { withDesign } from "storybook-addon-designs"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+
+import { VisuallyHidden } from ".."
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.a11y}/Visually Hidden`,
+  component: VisuallyHidden,
+  parameters: {
+    docs: {
+      description: {
+        component: 'import { VisuallyHidden } from "@kaizen/a11y";',
+      },
+    },
+  },
+  decorators: [withDesign],
```

Comment:
/nit this is just for embedding figma designs, which this component will never have, so you can remove it

Reply:
Thanks, I copy/pasted this setup from an existing one, not knowing what this does at all.